apollo-router/src/plugins/telemetry/config_new/router/selectors.rs

```diff
@@ -40,121 +40,124 @@ impl From<&RouterValue> for InstrumentValue<RouterSelector> {
 #[derive(Deserialize, JsonSchema, Clone, Debug, PartialEq)]
 #[serde(deny_unknown_fields, untagged)]
 pub(crate) enum RouterSelector {
```

Comment:
I alphabetized the order of enum variants. I also added a new one called `ResponseBody`.